docs/permissions/custom-rules.md

```diff
@@ -86,12 +86,12 @@ class TestPermissionPolicy implements PermissionPolicy {
       return createCatalogConditionalDecision(
         request.permission,
 -       catalogConditions.isEntityOwner(
--         user?.identity.ownershipEntityRefs ?? [],
+-         { claims: user?.identity.ownershipEntityRefs ?? [] },
```

Comment:
Potentially split these doc updates into a separate PR to ship them asap? 😁

Reply:
Harry beat me to it in https://github.com/backstage/backstage/pull/14196 💨